test/unit/db/seeds_test.rb

```diff
@@ -209,9 +214,9 @@ def teardown
   private
 
   def assert_expected_backend_api
-    assert (service = Account.tenants.first!.default_service)
+    assert(service = Account.tenants.first!.default_service)
```

Comment:
```suggestion
    assert (service = Account.tenants.first!.default_service)
```

Reply:
Ok but then I will do the same for the entire file. I don't mine `assert(` or `assert (` but it should be the same for the whole test :+1:

Reply:
Actually, we should avoid setting variables while invoking methods with parameters. (Kinda weird thing to say for Ruby but all right 😁)

Reply:
I disagree with that last part but that'ss a whole different topic :smile: